controller/rest/rest.go

```diff
@@ -1550,6 +1551,14 @@ func StartRESTServer() {
 	r.GET("/v1/scan/registry/:name/layers/:id", handlerRegistryLayersReport)
 	r.GET("/v1/scan/asset", handlerAssetVulnerability) // skip API document
 
+	// Sigstore Configuration
+	r.GET("/v1/scan/sigstore/global_config", handlerSigstoreGlobalConfigGet)
+	r.POST("/v1/scan/sigstore/global_config", handlerSigstoreGlobalConfigPost)
+	r.GET("/v1/scan/sigstore/verifier", handlerSigstoreVerifierGetAll)
+	r.GET("/v1/scan/sigstore/verifier/:name", handlerSigstoreVerifierGet)
+	r.POST("/v1/scan/sigstore/verifier/:name", handlerSigstoreVerifierPost)
```

Comment:
By convention, UI doesn't POST("/v1/.../:name") for creating a new named object.
POST("/v1/scan/sigstore/verifier) should be used for creating a new verifier.
It's PATCH("/v1/.../:name") for modifying an existing verifier.

Reply:
Ok. This will be implemented and resolved in a separate PR.